scripts/rename_classes.py

```
@@ -0,0 +1,252 @@
+                for change in change_list:
+                    print(change)
+                print()
+
+    print(f"Summary: {total_changes} changes across {modified_file_count} files (scanned {file_count} files)")
+    if dry_run:
+        print("Note: This was a dry run. No files were actually modified.")
+
+    return total_changes
+
+
+def main():
```

Ruff (0.8.2)

Missing return type annotation for public function `main`

Add return type annotation: `Optional[int]`

(ANN201)